Applications/Spire/Source/UiViewer/UiViewerWindow.cpp

```diff
@@ -94,6 +94,8 @@ UiViewerWindow::UiViewerWindow(QWidget* parent)
   add(make_icon_button_profile());
   add(make_text_box_profile());
   add(make_tooltip_profile());
+  add(make_vertical_scroll_bar_profile());
```

Comment:
It may be preferable to combine the ScrollBar test widgets into one and have the orientation be a settable property. Whoever is testing it will have to press rebuild to cause this to take effect, but at least it condenses the number of test widgets a little bit.